src/odemis/driver/technolution.py

```diff
@@ -113,11 +120,12 @@ def __init__(self, name, role, host, children={}, externalStorage={}, **kwargs):
         # debug because the API gives only minimal feedback on providing wrong parameters)
         self._calibrationParameters = None
 
-        self.asmApiPostCall("/config/set_external_storage?host=%s&user=%s&password=%s" %
-                            (urlparse(self.externalStorageURL.value).hostname,
-                             urlparse(self.externalStorageURL.value).username,
-                             urlparse(self.externalStorageURL.value).password), 204)
-        self.asmApiPostCall("/config/set_system_sw_name?software=%s" % name, 204)
+        # Comented out because not present on EA
```

Comment:
Though one what the flag should be. But since we want to uncomment this when it works a TODO seems indeed appropriate. I also fixed this for the other part in the test case in the same way.
@Sabrina85 Can you resolve the conversation if you think the fix is ok?

Reply:
Looks like I cannot resolve the issue - maybe because it is on your personal branch?
I am ok with the fix.